static/js/components/UserMenu_test.js

```diff
@@ -62,12 +59,14 @@ describe("UserMenu", () => {
       showUserMenu = showUserMenuValue
       const wrapper = renderUserMenu()
       if (showUserMenu) {
-        assert.isOk(wrapper.find(DropdownWithClickOutside).exists())
+        assert.isOk(wrapper.find(DropdownMenu).exists())
       } else {
-        assert.isNotOk(wrapper.find(DropdownWithClickOutside).exists())
+        assert.isNotOk(wrapper.find(DropdownMenu).exists())
       }
     })
   })
+
+  //
```

Comment:
Stray comment

Reply:
yeah, I put these in intentionally to force prettier to leave a blank line between tests declare with an array, just cause it annoys me, haha